xcm/v8/transfers_dynamic_dev.json

```diff
@@ -161,12 +161,6 @@
         "generalKey": "0x0001"
       }
     },
-    "KMA": {
```

Comment:
Remove it in v6/transfers as well since we use it as a foundation for legacy config construction